sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/encryption/DataEncryptionKeyProvider.java

```diff
@@ -15,10 +17,8 @@
      * @param encryptionAlgorithm Encryption algorithm that the retrieved key will be used with.
      * @return Data encryption key bytes.
```

Comment:
return type should be updated now here and other places.

Reply:
thanks for catching. I will fix javadoc in the next commit.

Reply:
done.